src/admin/components/UpgradeBanner.vue

```diff
@@ -1,23 +1,143 @@
 <template>
-    <div class="dokan-upgrade-bar" v-if="show && showUpgrade">
-        You're using <span>Dokan Lite</span>. To unlock more features, consider
-        <a target="_blank" rel="noopener" :href="upgradeURL">
-            Upgrading to Pro</a
-        >.
-
-        <div class="close-button" title="Dismiss the notice" @click="dismiss()">
-            &times;
+    <div class="dokan-promo-banner notice" v-if="show && showUpgrade">
+        <div class="thumbnail">
+            <svg viewBox="0 0 219 146" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Move this `svg` file in the root assets folder and import that here if it has the possibility to be used in other places.

Reply:
Have no possibility to reuse this svg